src/store/createQueryTransformResult.ts

```diff
@@ -375,14 +373,29 @@ function sendUpdate<T, S extends ObservableStore<any, any, any> & QueryTransform
 				update.movedInTracked.length || update.addedToTracked.length || update.removedFromTracked.length
 			)
 	)) {
-		let resolveInUpdate: Function | undefined = undefined;
-		state.inUpdate = new Promise((resolve) => {
-			resolveInUpdate = resolve;
-		});
 		state.observers.forEach(function(observer) {
-			observer.next(update);
+			if (isTracked(update)) {
```

Comment:
Not necessarily, but my concern was that if we don't make a separate copy for each observer then one observer could alter another observer's data. Which would really just be a different version of the issue this was trying to solve. 

I did a really rough test and if with 10,000 simple object items, each copy seems to come out to around 40kb of memory in Chrome and take about .5 milliseconds to make.

If that performance or memory usage is a concern this could potentially be a good use case for immutable.